doc/user/content/quickstarts/apps.md

```diff
@@ -0,0 +1,418 @@
+---
+title: "Data Applications"
+description: "Data Applications"
+menu:
+  main:
+    parent: 'quickstarts'
+    weight: 30
+    name: 'Data Applications'
+draft: true
+aliases:
+  - /demos/data-applications
+---
+
+In this quickstart, you'll build the analytics for an eCommerce sales dashboard that queries Materialize.
+At the end of this guide, you'll have the chance to explore an eCommerce data application querying against your code.
+
+The key concepts present in this quickstart will also apply to many other projects:
+
+* [Sources](https://materialize.com/docs/sql/create-source/load-generator/)
+* [Materialized views](https://materialize.com/docs/sql/create-materialized-view/)
+* [Subscriptions](https://materialize.com/docs/sql/subscribe/)
+
+### Prepare the environment
+
+1. Set up a [Materialize account.](/register)
+
+1. [Install psql and connect](https://materialize.com/docs/get-started/#connect) to Materialize.
+
+### Create the sources
+
+Materialize provides public Kafka topics and a Confluent Schema Registry for its users. The Kafka topics contain data from a fictional eCommerce and receive updates every second. You will use the data in them to build the analytics for the dashboard.
+
+1. In your `psql` terminal, create a new schema:
```

Comment:
Echoing the feedback in #16803:

> I'm thinking: should we promote the creation of a cluster here, rather than a schema? To double down on the concept.